lib/editor/components/pattern/TripPatternListControls.js

```diff
@@ -52,6 +52,7 @@ export default class TripPatternListControls extends Component<Props> {
       routeId: activeEntity.route_id,
       directionId: null,
       patternStops: [],
+      patternLocations: [],
```

Comment:
Sort attributes.

Reply:
This hasn't been changed in 4 years, weary of making unnecessary changes.